webrtc.html

```diff
@@ -3777,7 +3777,7 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
       SHA-256 hash.</p>
       <div>
         <pre class="idl">partial interface RTCPeerConnection {
-    static Promise&lt;RTCCertificate&gt; generateCertificate (AlgorithmIdentifier keygenAlgorithm);
+    static Promise&lt;RTCCertificate&gt; generateCertificate (RTCGenerateCertificateOptions keygenAlgorithm);
```

Comment:
Is it intentional that this will start rejecting with TypeError for `RTCCertificate.generateCertificate("string")`? The previous type was `object or DOMString`.